### PR TITLE
Feat is250Bits hint

### LINKED ISSUE #569: `Is250Bits` hint
https://github.com/starkware-libs/cairo-lang/blob/0e4dab8a6065d80d1c726394f5d9d23cb451706a/src/starkware/starknet/common/storage.cairo#L24C12-L24C55

https://github.com/lambdaclass/cairo-vm/blob/db7fff18c9c1312024ebf4119fcdfeba23657b12/vm/src/hint_processor/builtin_hint_processor/hint_code.rs#L101

https://github.com/starkware-libs/cairo-lang/blob/0e4dab8a6065d80d1c726394f5d9d23cb451706a/src/starkware/starknet/security/whitelists/0.10.3.json#L642

### PR DESCRIPTION
**Pull request by diegoTech14**
## Summary
Closes #569

### CODE REVIEW

**Comment by diegoTech14** [comment]
Hello, @TAdev0 finally I have found the way to solve the issue, please let me know if this looks good or there is something to change

File: pkg/hintrunner/zero/zerohint_math.go [hunk 1]
```diff
@@ -1189,3 +1189,46 @@ func createIsQuadResidueHinter(resolver hintReferenceResolver) (hinter.Hinter, e
 
 	return newIsQuadResidueHint(x, y), nil
 }
+
```
Comment (TAdev0) [inline pkg/hintrunner/zero/zerohint_math.go:1192]: can you add a comment that describes the hint here? you can take inspiration from other hints (for formatting)

File: pkg/hintrunner/zero/zerohint_math.go [hunk 2]
```diff
@@ -1189,3 +1189,46 @@ func createIsQuadResidueHinter(resolver hintReferenceResolver) (hinter.Hinter, e
...
+			if addr.Cmp(&utils.FeltUpperBound) < 0 {
+				is250Mv = memory.MemoryValueFromFieldElement(&utils.FeltOne)
+			} else {
+				is250Mv = memory.MemoryValueFromFieldElement(&utils.FeltZero)
+			}
+
+			return vm.Memory.WriteToAddress(&is250Addr, &is250Mv)
+		},
+	}
+}
```
Comment (TAdev0) [inline pkg/hintrunner/zero/zerohint_math.go:1220]: implementation looks good to me!

File: pkg/hintrunner/zero/zerohint_math_test.go [hunk 3]
```diff
@@ -902,5 +902,19 @@ func TestZeroHintMath(t *testing.T) {
 				check: varValueEquals("y", feltString("1484343478756640997457155271309092907848857951878936388435701743478603286656")),
 			},
 		},
+		"Is250Bits": {
```
Comment (TAdev0) [inline pkg/hintrunner/zero/zerohint_math_test.go:905]: can you make a case with `addr = 2**250` and `addr = 2**250 + 1`

**Review by TAdev0 — CHANGES_REQUESTED** [review]
Implementation of the hint looks good! left some minor comments.

Can you also move the `normalize_address.small.cairo` file to cairo_zero_hint_tests folder?

Your hint was required for integration test of this file, now it should work as expected.
You can run `make integration` locally for integrations

**Comment by diegoTech14** [comment]
Hey, @TAdev0 thanks for your comments, I'll work on that!

File: pkg/hintrunner/zero/zerohint_math_test.go [hunk 4]
```diff
@@ -902,5 +902,90 @@ func TestZeroHintMath(t *testing.T) {
...
+			{
+				operanders: []*hintOperander{
+					{Name: "is_250", Kind: uninitialized},
+					{Name: "addr", Kind: apRelative, Value: feltString("5")},
+				},
+				makeHinter: func(ctx *hintTestContext) hinter.Hinter {
+					return newIs250BitsHint(ctx.operanders["is_250"], ctx.operanders["addr"])
+				},
+				check: func(t *testing.T, ctx *hintTestContext) {
+					varValueEquals("is_250", feltInt64(1))
```
Comment (MaksymMalicki) [inline pkg/hintrunner/zero/zerohint_math_test.go:962]: you can immiadetely use `varValueEquals` in check, no need to use `func(t *testing.T, ctx *hintTestContext) {...`

**Review by MaksymMalicki — APPROVED** [review]

**Comment by MaksymMalicki** [comment]
Please resolve te conflicts and address the comments. Other than that minor comments, approve. Good job! 🚀

File: pkg/hintrunner/zero/zerohint_math_test.go (hunk 4 above)
Comment (diegoTech14) [reply pkg/hintrunner/zero/zerohint_math_test.go:962]: Oh, okay sir, I'm on it, thank you!

**Pull request closed without merging**